packages/edit-post/src/components/layout/style.scss

```diff
@@ -110,30 +110,3 @@
 		font-size: $default-font-size;
 	}
 }
-
-.edit-post-layout__scrollable-container {
-	// On mobile the main content (html or body) area has to scroll.
```

Comment:
Have you tested this on mobile?

Reply:
Cc @jasmussen. I'm not sure who originally added this.

Reply:
This class is not used anywhere.

Reply:
Oh wow, a lot of red here which I remember painstakingly adding. But yeah, if the class is not present anymore, this has either regressed long ago, or is completely unnecessary. I'll take a look!

Reply:
When (or in what commit) was the class removed?

Reply:
In this refactor #18044

Reply:
Ok, so it wouldn't have regressed long ago, the class was removed two days ago.

Reply:
It looks like the bulk of this behavior has moved to `editor-regions` and looks like this:

```
.edit-post-editor-regions__body {
	flex-grow: 1;
	display: flex;

	// On Mobile the header is fixed to keep HTML as scrollable.
	@include break-medium() {
		overflow: auto;
	}
}

.edit-post-editor-regions__content {
	flex-grow: 1;

	// On Mobile the header is fixed to keep HTML as scrollable.
	@include break-medium() {
		overflow: auto;
	}
}
```

I'll take a look at how it scrolls on iOS, which I think still needs things like -webkit-overflow-scrolling: touch;, and Firefox which I believe we added overscroll-behavior-y: none; for.

Reply:
Good news, this branch and master seems to work the same in iOS as it did before, with the correct element scrolling:

![master](https://user-images.githubusercontent.com/1204802/69408797-ef5cf400-0d07-11ea-94f9-ddd968e6b7fe.gif)

I think it might be worth it to restore some of these comments before the overflow rule quoted above, just because it's so easy to assume Mobile Safari works well when Safari works so well, when in fact Mobile Safari is horrendously bad and light years behind. The "wall of text" helps remind people to maybe test on the xcode simulator.

I'll be touching the top toolbar PR later today, I'll see if I can add it there.

Reply:
Looks like overscroll-behavior-y: none isn't necessary either, scrolling works in <600px in chrome and Firefox, as was the reason for the rule in the first place:

https://github.com/WordPress/gutenberg/pull/10539